src/layouts/legacyLoggedIn.vue

```diff
@@ -133,9 +133,11 @@ export default {
 		sidebarItems() {
 			const sidebarItems = this.sidebarBaseItems.filter((item) => {
 				const hasRequiredPermission =
+					this.user &&
 					this.user.permissions &&
 					this.user.permissions.includes(item.permission);
```

Comment:
I think you can use optional chaining in this repo 🎉 At least I hope so and did it somewhere else. 
```suggestion
hasRequiredPermission =
					this.user?.permissions &&
					this.user?.permissions.includes(item.permission);
```

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining

Reply:
![](https://media1.giphy.com/media/ue1GO5swPdORq/giphy.gif?cid=790b761126bcfc975b451bb51063da207026f644bd77e922&rid=giphy.gif)